docs/source/scheduled-queries.rst

```diff
@@ -0,0 +1,297 @@
+    """,
+        params=['utcdatehour_minus1hour'],
+        tags=['sample']
+    )
+
+* ``name`` - (str) The name of this query. This name is published in the final result, and is useful when writing rules.
+* ``description`` - (str) Description of this query. This is published in the final result.
+* ``query`` - (str) A template SQL statement sent to Athena, with query parameters identified ``{like_this}``.
+* ``params`` - (list[str]) A list of query parameters to pass to the query string. These have special values that are calculated at runtime, and are interpolated into the template SQL string.
+* ``tags`` - (list[str]) Tags required by this query to be run. The simplest way to use this is to put the **Query pack name** into this array.
+
+
+
+Writing Rules for StreamQuery
+-----------------------------
+
+Classifier Schema
+`````````````````
+We provide an out-of-box sample schema for scheduled query v1.0.0 results. It is located at ``conf/schemas/streamquery.json``.
+
+
+What does a scheduled query result look like?
+`````````````````````````````````````````````
+Below is an example of what StreamAlert may receive as a result from a scheduled query.
+
+.. code-block:: json
+
+    {
+        "streamquery_schema_version": "1.0.0",
+        "execution": {
+            "name": "query_name_goes_here",
+            "description": "This is an example",
+            "query": "SELECT *\nFROM my_database.my_table\nWHERE dt = '2020-01-01-01' LIMIT 10",
+            "query_parameters": {
+                "dt": "2020-01-01-01"
+            },
+            "data_scanned_in_bytes": 4783293,
+            "execution_time_ms": 12374,
+            "tags": [ "query_pack_1" ],
+            "query_execution_id": "123845ac-273b-ad3b-2812-9812739789",
+            "console_link": "https://console.amazonaws.com/athena/somethingsomething",
+        },
+        "data": {
+            "headers": [
+                "username",
+                "time"
+            ],
+            "rows": [
+                {
+                    "username": "bob",
+                    "time": 1,
+                },
+                {
+                    "username": "sally",
+                    "time": 2,
+                },
+                {
+                    "username": "joe",
+                    "time": 3,
+                },
+            ],
+            "count": 3,
+        },
+    }
+
+Because the **data** of each query may be different it is generally advisable to write a StreamAlert
+matcher on the ``execution.name`` value of the data, first. The rest is up to you!
+
+
+Deployment
+----------
+Deploying the various components of scheduled_queries is easy.
+
+Building the Step Function, Lambda, and Query Packs
+```````````````````````````````````````````````````
+
+Anytime you change the configured query packs, you will need to run this to update the AWS Resources.
+
+.. code-block:: bash
+
+    % ./manage.py built -t scheduled_queries
```

Comment:
@ryandeivert  recently updates the doc to use `python manage.py` instead of `% ./manage.py` or `$ ./manage.py`. Let's keep it consistent, otherwise @ryandeivert will send PR to address this in the future.

Reply:
Maybe use `Find All` and `Repleace All` to save time.

Reply:
I may miss something, do we mention the proper permissions for the admin users to built and deploy `scheduled_queries` in the doc?

Reply:
Yeah, I have an internal PR that was merged on vpc-tf for that; numbered 623